docs/recipe_user_guide/recipes.md

```diff
@@ -26,10 +26,18 @@ to the public Pangeo Forge {doc}`../cloud_automation_user_guide/recipe_box`, whe
 
 To write a recipe, you must start from one of the existing recipe classes.
 Recipe classes are based on a specific data model for the input files and target dataset format.
-Right now, there is only one recipe class implemented:
+Right now, there are two distinct recipe classes implemented.
+In the future, we may add more.
+
+TODO add rubric for choosing a recipe class.
```

Comment:
Did you intend to leave this note in?

Reply:
Yes, it's a note to self to do this. I added #306 to track it specifically so we can not worry about it here.